### DISCUSSION TITLE
which physical file will artisan save when changing the extra device information under device assignment

### DISCUSSION THREAD

**Discussion by mansonchan888** [artisan-roaster-scope/artisan#1132] [Q&A]
I am trying to locate the external physical file that Artisan will save into when I changed device assignment / extra devices / virtual information.
Can anyone help me?
that reason I am asking the system keep bring back the old setting, I want to find out where it pick up the information from

**Comment by bigl00z3** [comment]
If you are on a Mac, the apps stores documents in `~/Library/Application\ Support/artisan-scope/Artisan`

If you are on Windows it's in your `user/AppData/local `(check the exact path, I don't have my windows computer with me.

But that sounds weird. Is your account on the computer without admin rights? It should not behave like this. Do you have right access to the folder?

Nico
